ssc/cmod_pvwattsv5.cpp

```diff
@@ -532,7 +532,7 @@ class cm_pvwattsv5 : public cm_pvwattsv5_base
 					p_aoi[idx] = (ssc_number_t)aoi;
 
 					double shad_beam = 1.0;
-					if (shad.fbeam(hour, solalt, solazi, jj, step_per_hour))
+					if (shad.fbeam(hour, wf.minute, solalt, solazi))
```

Comment:
This had to be reverted before as all tests were failing with row translation calculations in the shading_factors classes. The results of the test were close but off and the old method had to be used. Did the shading tests for pvsam_v1 and pvwatts and for the swh pass with this change? Also, are there shading tests for swh and pvwatts?

Reply:
This is different logic than was failing before. All shading tests pass with this logic. I don't know that there are tests for swh and pvwatts shading.